jaxrs/src/main/java/se/fortnox/reactivewizard/ExceptionHandler.java

```diff
@@ -67,7 +68,7 @@ public ExceptionHandler() {
             webException = new WebException(HttpResponseStatus.BAD_REQUEST, "invalidjson", throwable.getMessage());
         } else if (throwable instanceof WebException) {
             webException = (WebException)throwable;
-        } else if (throwable instanceof ClosedChannelException) {
+        } else if (throwable instanceof ClosedChannelException || throwable instanceof AbortedException) {
             LOG.debug("ClosedChannelException: {} {}", request.method(), request.uri(), throwable);
```

Comment:
Maybe change the debug message to something like: "Inbound connection has been closed"? 

What is the difference between closedChannel and abortedexception?

Reply:
Fixed. 
With RxNetty we got the raw ClosedChannelException.
Now it comes in a AbortedException as the cause